nilearn/plotting/html_surface.py

```diff
@@ -0,0 +1,584 @@
+        See http://nilearn.github.io/manipulating_images/input_output.html
+
+    mesh : str or dict, optional.
+        if 'fsaverage5', use fsaverage5 mesh from nilearn.datasets
+        if a dictionary, keys should be 'infl_left', 'pial_left', 'sulc_left',
+        'infl_right', 'pial_right', 'sulc_right', containing inflated and
+        pial meshes, and sulcal depth values for left and right hemispheres.
+
+    threshold : int, optional
+        int in [0, 100]: percentage of values to be thresholded.
+
+    cmap : str or matplotlib colormap, optional
+
+    black_bg : bool, optional (default=False)
+
+    embed_js : bool, optional (default=True)
+        if True, jquery and plotly are embedded in resulting page.
+        otherwise, they are loaded via CDNs.
+
+    Returns
+    -------
+    HTMLDocument : html page containing a plot of the stat map.
+
+    """
+    info, colors = full_brain_info(
+        stat_map=stat_map, mesh=mesh, threshold=threshold,
+        cmap=cmap, black_bg=black_bg)
+    return _fill_html_template(info, colors, embed_js=embed_js)
+
+
+def view_surf(surf_mesh, surf_map=None, bg_map=None, threshold=None,
+              cmap=cm.cold_hot, black_bg=False, symmetric_cmap=True,
+              embed_js=True):
+    """
+    Insert a surface plot of a surface map into an HTML page.
+
+    Parameters
+    ----------
+    surf_mesh: str or list of two numpy.ndarray
+        Surface mesh geometry, can be a file (valid formats are
+        .gii or Freesurfer specific files such as .orig, .pial,
+        .sphere, .white, .inflated) or
+        a list of two Numpy arrays, the first containing the x-y-z coordinates
+        of the mesh vertices, the second containing the indices
+        (into coords) of the mesh faces.
+
+    surf_map: str or numpy.ndarray, optional.
+        Data to be displayed on the surface mesh. Can be a file (valid formats
+        are .gii, .mgz, .nii, .nii.gz, or Freesurfer specific files such as
+        .thickness, .curv, .sulc, .annot, .label) or
+        a Numpy array
+
+    bg_map: Surface data, optional,
+        Background image to be plotted on the mesh underneath the
+        surf_data in greyscale, most likely a sulcal depth map for
+        realistic shading.
+
+    threshold : int, optional
+        int in [0, 100]: percentage of values to be thresholded.
+
+    cmap : str or matplotlib colormap, optional
+        You might want to change it to 'gnist_ncar' if plotting a
+        surface atlas.
+
+    black_bg : bool, optional (default=False)
+
+    symmetric_cmap : bool, optional (default=True)
+        Make colormap symmetric (ranging from -vmax to vmax).
+        Set it to False if you are plotting a surface atlas.
+
+    embed_js : bool, optional (default=True)
+        if True, jquery and plotly are embedded in resulting page.
+        otherwise, they are loaded via CDNs.
+
+    Returns
+    -------
+    HTMLDocument : html page containing a plot of the stat map.
```

Comment:
Same comment on the return value.

Maybe we should call this object "HTMLView"?

Reply:
but it contains an html document; it's not clear to me what HTMLView means